### PR TITLE
8.0 test for crm_compassion

### PR DESCRIPTION
**Pull request by albertsh**
- change test_crm_compassion on 8.0 version

### CODE REVIEW

**Comment by ecino** [comment]
PEP8 errors. Please verify more carefully before sending pull request...

File: contract_compassion/tests/test_base_module.py [hunk 1]
```diff
@@ -20,8 +20,8 @@ class test_base_module(test_base_contract):
 
     def setUp(self):
         # Retrieve of income account
-        account_obj = self.env['account.account']
-        property_account_income = account_obj.search([
+        super(test_base_module, self).setUp()
```
Comment (ecino) [inline contract_compassion/tests/test_base_module.py]: Right before you told me you would not call super from here and now you add it. Please explain.

**Comment by ecino** [comment]
Remove all pdb

File: contract_compassion/tests/test_base_module.py (hunk 1 above)
Comment (albertsh) [inline contract_compassion/tests/test_base_module.py]: if i don't add it, i have an error saying that test_sponsorship_compassion or test_crm_compassion object have no env attribute when i do a simple self.env[something]

Comment (albertsh) [inline contract_compassion/tests/test_base_module.py]: oh my bad i didn't understood the question well, i thought the question was why i haven't inherit from test_recurring_contract

**Comment by ecino** [comment]
:+1:

**Pull request merged**